packages/design-system/src/components/web-components/ds-step-list/ds-step-list.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import type { Meta } from '@storybook/react';
+import WebComponentDocTemplate from '../../../../../../.storybook/docs/WebComponentDocTemplate.mdx';
+import { webComponentDecorator } from '../storybook';
+import { stepListStepData } from '../../StepList/StepList.stories';
+import './ds-step-list';
+
+const meta: Meta = {
+  title: 'Web Components/ds-step-list',
+  decorators: [webComponentDecorator],
+  parameters: {
+    docs: {
+      page: WebComponentDocTemplate,
+      description: {
+        component: `For information about how and when to use this component, [refer to its full documentation page](https://design.cms.gov/components/step-list/).
+
+**Note:** The \`ds-step-list\` is based on the existing functionality of our React component \`StepList\`. Unlike \`StepList\`, which uses an optional \`component\` prop to render custom components for link elements—defaulting to an \`<a>\` tag if not provided—the \`ds-step-list\` does not support the \`component\` attribute. Instead, all link elements in \`ds-step-list\` will default to standard anchor (\`<a>\`) tags to simplify the API and ensure consistent behavior.`,
```

Comment:
Is it worth saying something here about the missing onClick handler, as well?

Reply:
Good call out! Will do!